app/client/views/home/home.html

```diff
@@ -13,22 +13,22 @@
 
 			<div class="collapse navbar-collapse" id="navbarSupportedContent1">
 				<ul class="nav navbar-nav float-right text-right ml-auto">
-					<li class="nav-item">
+					<li class="nav-item" data-toggle="collapse" data-target=".navbar-collapse">
 						<a class="page-scroll" href="#about">About</a>
 					</li>
-					<!--<li class="nav-item">
+					<!--<li class="nav-item" data-toggle="collapse" data-target=".navbar-collapse">
 						<a class="page-scroll" href="#schedule">Schedule</a>
 					</li>-->
-					<li class="nav-item">
+					<li class="nav-item" data-toggle="collapse" data-target=".navbar-collapse">
 						<a class="page-scroll" href="#sponsors">Sponsors</a>
 					</li>
-					<li class="nav-item">
+					<li class="nav-item" data-toggle="collapse" data-target=".navbar-collapse">
 						<a class="page-scroll" href="#faq">FAQ</a>
 					</li>
-					<li class="nav-item">
+					<li class="nav-item" data-toggle="collapse" data-target=".navbar-collapse">
 						<a class="page-scroll" href="#contact">Contact</a>
 					</li>
-					<li class="nav-item">
+					<li class="nav-item" data-toggle="collapse" data-target=".navbar-collapse">
```

Comment:
Just to be sure, is this collapsing behaviour wanted on desktop level? Because every time I click on the desktop navbar it does this weird eye open/eye close animation on the navbar itself.

Reply:
... unintentional. Do we need to fix it? The fixes are quite ugly .. and the eye blinking is charming? xD